lib/developer_portal/app/controllers/developer_portal/admin/account/passwords_controller.rb

```diff
@@ -46,13 +46,19 @@ def redirect_to_request_password(error_message)
   end
 
   def buyer
-    @buyer ||= @provider.buyers.build
+    @buyer ||= @provider.buyers.build.tap do |account|
```

Comment:
Is `.tap` needed here? I think it would work just with `.build do |account|`

Reply:
Indeed we can remove `tap` as `build` already returns newly built object.

Reply:
Removed!